app/controllers/login_controller.php

```diff
@@ -0,0 +1,35 @@
+<?php
+class LoginController extends AppController 
+{
+    public function login()
+    {
+        $check = Param::get('call', false);
```

Comment:
What is this param for po? Since it's a static value, wouldn't it will always be true? But if this is really needed, please make the param name more clear po. (example: is_check) :bow: